src/media_player.py

```diff
@@ -0,0 +1,81 @@
+"""
+Media-player entity functions.
+
+:copyright: (c) 2023 by Unfolded Circle ApS.
+:license: Mozilla Public License Version 2.0, see LICENSE for more details.
+"""
+
+import logging
+from typing import Any
+
+from ucapi import EntityTypes, MediaPlayer, StatusCodes
+from ucapi.media_player import Commands, DeviceClasses, Options
+
+import tv
+from config import AtvDevice, create_entity_id
+from profiles import Profile
+
+_LOG = logging.getLogger(__name__)
+
+
+class AndroidTVMediaPlayer(MediaPlayer):  # pylint: disable=too-few-public-methods
+    """Representation of a AndroidTV Media Player entity."""
+
+    def __init__(self, device_config: AtvDevice, device: tv.AndroidTv, profile: Profile):
+        """Initialize the class."""
+        # pylint: disable = R0801
+        _LOG.debug("[%s] AndroidTVMediaPlayer init", device_config.address)
+        self._device = device
+        self._device_config = device_config
+        self._profile = profile
+
+        entity_id = create_entity_id(device_config.id, EntityTypes.MEDIA_PLAYER)
```

Comment:
Changing the entity identifier of the existing media-player entity is a breaking change we can't allow. Otherwise all existing user configurations won't work anymore and have to be redone (including activities, macros).